src/Dfc.CourseDirectory.Web/appsettings.json

```diff
@@ -79,5 +79,9 @@
     "RetrieveAddressBaseUrl": "https://services.postcodeanywhere.co.uk/PostcodeAnywhere/Interactive/RetrieveById/v1.30/json3.ws?"
   },
   "ProviderAzureSearchIndexName": "provider",
-  "LarsAzureSearchIndexName": "lars"
+  "LarsAzureSearchIndexName": "lars",
+  "GetAddressSettings": {
+    "UseGetAddress": true,
```

Comment:
Assuming this can be overridden simply by setting environment variable `GetAddressSettings::UseGetAddress` to `false`?